pages/common/semgrep.md

```diff
@@ -0,0 +1,17 @@
+# semgrep
+
+> Perform static source code analysis (SAST) to detect security vulnerabilities.
+> It is a fast SAST tool aiming to minimize false positives.
```

Comment:
I don't like the `fast` part. We try to have objective command description. You can leave the `minimiza false positives` part.
```suggestion
> It is a SAST tool aiming to minimize false positives.
```

Reply:
I don not want to drop the "fast" part completely, as it is really an important design part and the speed difference to other SAST tool can be multiple orders of magnitude.
But I understand your concern, i think this rewording should be fine:

```suggestion
> It is a SAST tool aiming to minimize false positives and being fast enough to be used in CI pipelines.
```